Python/Algorithms/aditya/traversal.py

```diff
@@ -0,0 +1,123 @@
+from collections import deque
```

Comment:
Again, use the queue that you have created

Reply:
OK , have changed to use my own implementation of stack and queue .